### ISSUE TITLE
exec: Fail to find script file referenced with relative paths

### ISSUE THREAD

**Issue by ttimasdf** [adnanh/webhook#316]
There are already some threads addressing this issue: #178, #179, #214. They have different solutions but I think they're just sort of "hacks" and I believe this is a bug.

I take the config as an example from #178:
```
$ cat hooks.json 
[
  {
    "id": "redeploy-webhook",
    "execute-command": "./redeploy.sh",
    "command-working-directory": "./webhook"
  }
]
$ cat redeploy.sh 
#!/bin/sh
echo 'success'
```
If I take the cwd of `webhook` program itself as `.`. From the config, the script file should be located and probed at `./webhook/redeploy.sh`.

But from these lines and according to [Godoc for exec package](https://golang.org/pkg/os/exec/#LookPath)
https://github.com/adnanh/webhook/blob/0aa7395e2191278c090ecadc94472f6205cd9a01/webhook.go#L326-L332

Instead it looks for `./redeploy.sh` (invoke `exec.LookPath` to search relatively from cwd of `webhook` program instead of the `command-working-directory` in config file) and result in an error, where I believe is the root cause of this issue.

Meanwhile, I executed `env` and `pwd` in webhook finding that the cwd property (I don't know where it resides in process context) is right, but the `$PWD` environmental variable is wrong for the subprocesses.

**Comment by adnanh** [comment]
Correct, this is a bug, we should invoke `LookPath` only if the the `h.ExecuteCommand` is not an actual path (relative, or absolute).

**Comment by adnanh** [comment]
This should be fixed now in webhook 2.6.10
